php/ng/fpm/init.sls

```diff
@@ -5,14 +5,34 @@ include:
   - php.ng.fpm.service
   - php.ng.fpm.pools
 
+{% set pillar_php_ng_version = salt['pillar.get']('php:ng:version', '7.0') %}
 extend:
   php_fpm_service:
     service:
       - watch:
+    {% if pillar_php_ng_version is iterable and pillar_php_ng_version is not string %}
```

Comment:
now I'm not sure how this handles when 7.3 as a float is used. Is setting the version without quotes supported by the existing version of the formula ?